### ISSUE TITLE
CNN_KAN training to achieve ~95% test accuracy

### ISSUE THREAD

**Issue by sgchr273** [jakariaemon/CNN-KAN#2]
Hi,

Thanks for the great work and posting the code for KAN_CNN. I am using this code for an OOD task where I am training model on CIFAR10 dataset while evaluating it on SVHN just see how much differentiating capability did KAN_CNN obtain? For that purpose I wanted to train both the CNN and KAN_CNN to at-least 90% test accuracy to draw some conclusions. Can you guide on how can I achieve higher accuracy for KAN_CNN? Is it playing with learning_rate or weight_decay or something else I need to keep in mind.

Thanks in advance
